selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/YamlFileEditingTest.java

```diff
@@ -60,6 +68,7 @@
           + "    app: che\n"
           + "    template: che\n"
           + "  name: che";
+  private List<String> symbols = Arrays.asList("apiVersionsymbols (194)", "kind", "metadata");
```

Comment:
`symbols` variable looks like a constant, and "apiVersionsymbols (194)" is more then one symbol.
So, I would recommend to remake variable `symbols` to constant `EXPECTED_GO_TO_SYMBOL_ALTERNATIVES`